.eslintrc.json

```diff
@@ -53,6 +56,7 @@
         "comma-spacing": ["error", { "before": false, "after": true }],
         "computed-property-spacing": ["error", "never"],
         "func-call-spacing": ["error", "never"],
+        "function-paren-newline": ["error", "multiline-arguments"],
```

Comment:
`search.js`:

```javascript
this._onKeyDownIgnoreKeys = new Map([
    ['ANY_MOD', new Set([
        'Tab', 'ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight', 'PageDown', 'PageUp', 'Home', 'End',
        'F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'F7', 'F8', 'F9', 'F10',
        'F11', 'F12', 'F13', 'F14', 'F15', 'F16', 'F17', 'F18', 'F19', 'F20',
        'F21', 'F22', 'F23', 'F24'
    ])],
    ['Control', new Set(['C', 'A', 'Z', 'Y', 'X', 'F', 'G'])],
    ['Meta', new Set(['C', 'A', 'Z', 'Y', 'X', 'F', 'G'])],
    ['OS', new Set()],
    ['Alt', new Set()],
    ['AltGraph', new Set()],
    ['Shift', new Set()]
]);
```

Do you consider this something that can be done even if function argument newline consistency is enforced?

Reply:
All of that is inside `[]` brackets, so that's all fine. I consider functions to be a special case, since they typically have a limited and explicit number of arguments.

Whenever I'm doing multiline function arguments, I prefer to have one per line. There was only the one place in the project where this wasn't being done, so it felt consistent enough to enable the rule.

Reply:
Yeah I agree that this is a bit different from functions because in functions the position usually has a meaning and the amount is limited like you said. Here it's just about grouping related things together. Speaking of grouping, I think it would make more sense if it was grouped by 12 because that's the amount of function keys on the keyboard.